serverlessworkflow/modules/ROOT/pages/core/custom-functions-support.adoc

```diff
@@ -329,14 +329,14 @@ You can declare a {product_name} custom function using the Knative service name,
     {
       "name": "greet", <1>
       "type": "custom", <2>
-      "operation": "knative:custom-function-knative-service" <3>
+      "operation": "knative:services.v1.serving.knative.dev/custom-function-knative-service?path=/plainJsonFunction", <3>
```

Comment:
Same here. We are saying that it will send a POST request to the root path a few lines below. There is also no description about the path parameter, whether it is mandatory or the default is `/`.

Reply:
Fixed. It's not the root path. The path is `/plainJsonFunction`.
Added information about the path not being mandatory.